jgiven-html-app/src/js/util.js

```diff
@@ -225,4 +225,17 @@ export function replaceArguments(string, enumArray, nameArray) {
         }
     }
     return result.join("");
+}
+
+export function insertThumbnailPath(path) {
```

Comment:
I would rather call this function `getThumbnailPath`. Maybe you could also add a short documentation.